examples/build.gradle

```diff
@@ -5,32 +5,31 @@ buildscript {
         google()
     }
     dependencies {
-        classpath("com.google.protobuf:protobuf-gradle-plugin:0.8.18")
+        classpath("com.google.protobuf:protobuf-gradle-plugin:${protobufGradlePluginVersion}")
     }
 }
 
 plugins {
     id "java"
-    id "com.google.protobuf" version "0.8.18"
+    id "java-library"
+    id "com.google.protobuf"
     id "idea"
     id "application"
     id "distribution"
 }
 
-def grpcVersion = "1.46.0"
-def protobufVersion = "3.20.1"
-def protocVersion = protobufVersion
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
     // gRPC Utils project
+    api platform(project(":dependencies"))
```

Comment:
Why don't use `implementation platform(project(":dependencies"))` ?

Reply:
I think it could be removed instead because the dependency of `implementation project(path: ':grpc-utils')` gives already it with transitivity, isn't it?

Reply:
I removed it, even though for me it may be worth to keep it, in order to explicitly depend on the file where the dependencies are defined.

Reply:
In this case, if you want to force it explicit, to be coherent, I think you need to put `implementation platform(project(":dependencies"))` instead of `api platform(project(":dependencies"))` in `grpc-utils/build.gradle`, with it `implementation platform(project(":dependencies"))` will be mandatory here, but I don't test it

Reply:
done